examples/web-components-kitchen-sink/package.json

```diff
@@ -47,26 +48,30 @@
     "@storybook/source-loader": "portal:../../lib/source-loader",
     "@storybook/theming": "portal:../../lib/theming",
     "@storybook/ui": "portal:../../lib/ui",
-    "@storybook/web-components": "portal:../../app/web-components"
+    "@storybook/web-components": "portal:../../app/web-components",
+    "typescript": "4.2.4"
```

Comment:
Set a fixed version to avoid compat' issue with the Yarn 2 version we are using for now